include/seqan3/alphabet/composition/union_composition.hpp

```diff
@@ -507,14 +478,6 @@ class union_composition
      * An array which contains the prefix sum over all
      * alternative_types::value_size's.
      *
-     * ```cpp
-     * constexpr std::array partial_sum = union_composition<dna5, gap, dna5>::partial_sum_sizes; // not working; is protected
-     * assert(partial_sum.size() == 4);
-     * assert(partial_sum[0] == 0);
-     * assert(partial_sum[1] == 4);
-     * assert(partial_sum[2] == 5);
-     * assert(partial_sum[3] == 10);
-     * ```
```

Comment:
I think @h-2 said we don't need these examples (https://github.com/seqan/seqan3/pull/389#discussion_r213306914) because I guess these things are protected so they're not used?